web-ui/src/app/pages/home-page/home-content/home-content.component.html

```diff
@@ -55,18 +55,21 @@
             </div>
         </div>
     </div>
-    <div class="column">
-        <div class="card">
-            <header class="card header">
-                <a [href]="uploadUrl" class="card-header-title">Corpus Upload</a>
-            </header>
-            <div class="card-content">
-                <div class="content">
-                    <p>Allows you to upload your own corpus to GrETEL.</p>
+    <!--
```

Comment:
Denk dat hiervoor een optie moet worden toegevoegd

Reply:
Oeps, die is per ongelijk kwijt geraakt! Het was niet bepaald de bedoeling dat upload er helemaal uit zou gaan.

Reply:
Is teruggebracht in de laatste versie. Wellicht zou er een switch op kunnen als `uploadProvider` niet is geconfigureerd, maar dit is nog niet gedaan. Is dat iets dat je sowieso zou willen zien, of was het meer als alternatief voor het compleet verwijderen bedoelt?